app/assess/routes.py

```diff
@@ -297,6 +299,7 @@ def score(
         flag_status=flag_status,
         assessment_status=assessment_status,
         is_flaggable=is_flaggable(flag_status),
+        guidance_link=state.fund_guidance_url,
```

Comment:
We are already passing state, so do we need to do this?

Could we maybe just use `state.fund_guidance_url` on the template?

Reply:
yup you're right, I have now passed the state through to the scores_justification.html instead